src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.Log.cs

```diff
@@ -174,6 +174,12 @@ private static class Log
             private static readonly Action<ILogger, string, Exception> _removingHandlers =
                LoggerMessage.Define<string>(LogLevel.Debug, new EventId(58, "RemovingHandlers"), "Removing handlers for client method '{MethodName}'.");
 
+            private static readonly Action<ILogger, string, Exception> _sendingMessageGeneric =
+                LoggerMessage.Define<string>(LogLevel.Debug, new EventId(59, "SendingMessageGeneric"), "Sending {MessageType} message.");
```

Comment:
@davidfowl Have we decided what constitutes a "breaking change" with Logging messages? In theory, we could make `SendingMessage` and `SendingMessageGeneric` the same EventID, but with different delegates, one that takes an Invocation ID and one that does not.